styles.css

```diff
@@ -0,0 +1,37 @@
+.fas {
+  transition: transform 0.5s;
+}
+
+.rotate {
+  transform: rotate(90deg);
+}
+
+.nav-link:hover,
+.show-answer:hover {
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.fas:hover {
+  cursor: pointer;
+}
+
+.vertical-center {
```

Comment:
Check out Bootstrap's `flexbox` utilities.

Reply:
I eventually was able to get this working acceptably, but still ended up having to write custom CSS to set the height of the parent container equal to the viewport (it was that or set html, body, div {height: 100%} which had terrible side-effects), in addition to having to explicitly set the width of the card to take up 100% of the `flexbox` container.  Is there a cleaner way to do this?